Userland/Libraries/LibWeb/Crypto/SubtleCrypto.cpp

```diff
@@ -22,13 +22,14 @@ JS::Promise* SubtleCrypto::digest(String const& algorithm, JS::Handle<JS::Object
     // 1. Let algorithm be the algorithm parameter passed to the digest() method.
 
     // 2. Let data be the result of getting a copy of the bytes held by the data parameter passed to the digest() method.
-    auto data_buffer = Bindings::IDL::get_buffer_source_copy(*data.cell());
-    if (!data_buffer.has_value()) {
+    auto data_buffer_or_error = Bindings::IDL::get_buffer_source_copy(*data.cell());
+    if (data_buffer_or_error.is_error()) {
         auto* error = wrap(wrapper()->global_object(), DOM::OperationError::create("Failed to copy bytes from ArrayBuffer"));
         auto* promise = JS::Promise::create(global_object);
         promise->reject(error);
         return promise;
     }
+    auto& data_buffer = data_buffer_or_error.value();
```

Comment:
Let me know if we want to use `release_value()` here. Seems appropriate to take a reference by using `value()` since we are making a copy of it further down.